packages/sanity/src/core/form/studio/assetSource/DefaultSource.tsx

```diff
@@ -207,6 +233,18 @@ const DefaultAssetSource = function DefaultAssetSource(
       onClose={handleClose}
       __unstable_autoFocus={hasResetAutoFocus}
     >
+      {showAcceptMessage && (
+        <Card tone="primary" marginTop={4} marginX={4} padding={[3, 3, 4]} border radius={2}>
+          <Flex gap={[3, 4]} align="center">
+            <Text>
+              <InfoOutlineIcon />
+            </Text>
+            <Text size={1}>
+              Only showing assets of accepted types: <b>{acceptTypes}</b>
```

Comment:
After bringing up in the Standup, it was decided to go with the printing of the types accepted as a first go.